api/src/pcapi/routes/adage_iframe/serialization/adage_authentication.py

```diff
@@ -59,3 +61,15 @@ def get_redactor_information_from_adage_authentication(
     except ValidationError:
         raise MissingRequiredRedactorInformation()
     return redactor_information
+
+
+def get_offers_count(authenticated_information: AuthenticatedInformation) -> int:
+    offer_query = (
+        educational_models.CollectiveOffer.query.join(
+            educational_models.EducationalInstitution, educational_models.CollectiveOffer.institution
+        )
+        # .filter(educational_models.CollectiveOffer.isBookable == True)
+        .filter(educational_models.EducationalInstitution.institutionId == authenticated_information.uai)
+    )
+    offer_count = len([count for count in offer_query if count.isBookable == True])
```

Comment:
[style] Il est plus idiomatique en Python d'omettre `== True` : 

```suggestion
    offer_count = len([count for count in offer_query if count.isBookable ])
```